docs/types/reference-types.rst

```diff
@@ -550,3 +550,9 @@ members of the local variable actually write to the state.
 Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
+
+.. note::
+    In previous solidity versions, assigments which create memory-structs that contain a mapping
```

Comment:
Let's clarify that. The change was made in #9146, which was included in Solidity 0.7.0.

```suggestion
    Until Solidity 0.7.0, assigments which create memory-structs that contain a mapping
```